### PR TITLE
Reader: move email settings popover to new block

### PR DESCRIPTION
**Pull request by bluefuton**
So that we can reuse the Email Settings popover functionality elsewhere (specifically the stream - see https://github.com/Automattic/wp-calypso/issues/15457), this PR relocates the component responsible from inside `reader-subscription-list-item` to a new block, `reader-email-settings`.

Styles have been migrated and classes renamed. I've also added a couple of missing translated strings.

### To test 

Ensure that there are no visual or functional regressions in the email settings popover on http://calypso.localhost:3000/following/manage.

You can also check out the new Devdocs page at:

http://calypso.localhost:3000/devdocs/blocks/reader-email-settings

### CODE REVIEW

File: client/blocks/reader-email-settings/index.jsx [hunk 1]
```diff
@@ -129,8 +131,8 @@ class ReaderEmailSubscriptionSettingsPopout extends Component {
 									{ translate( 'Weekly' ) }
 								</ControlItem>
 							</SegmentedControl> }
-						<div className="reader-subscription-list-item__email-popout-toggle">
-							New comments
+						<div className="reader-email-settings__popout-toggle">
+							{ translate( 'New comments' ) }
```
Comment (samouri) [inline client/blocks/reader-email-settings/index.jsx]: nice!

File: client/blocks/reader-subscription-list-item/index.jsx [hunk 2]
```diff
@@ -109,11 +108,14 @@ function ReaderSubscriptionListItem( {
 							className="reader-subscription-list-item__link"
 							onClick={ recordTitleClick }
 						>
-							{ ' ' }{ siteTitle }{ ' ' }
+							{' '}
```
Comment (samouri) [inline client/blocks/reader-subscription-list-item/index.jsx]: we can just get rid of these kludges I think

File: client/blocks/reader-email-settings/docs/example.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const exampleSiteId = 70135762; // Longreads
+
+export default class ReaderEmailSettingsExample extends React.Component {
+	static displayName = 'ReaderEmailSettings';
+
+	render() {
+		return (
+			<Card>
+				<SyncReaderFollows key="syncReaderFollows" />
```
Comment (samouri) [inline client/blocks/reader-email-settings/docs/example.jsx]: do we need the key here?

File: client/devdocs/design/blocks.jsx [hunk 4]
```diff
@@ -92,18 +92,16 @@ export default React.createClass( {
 	render() {
 		return (
 			<Main className="design">
-				{
-					this.props.component
+				{ this.props.component
 					? <HeaderCake onClick={ this.backToComponents } backText="All Blocks">
-						{ slugToCamelCase( this.props.component ) }
-					</HeaderCake>
+							{ slugToCamelCase( this.props.component ) }
```
Comment (samouri) [inline client/devdocs/design/blocks.jsx]: i have a feeling this won't pass eslint :(

**Comment by samouri** [comment]
Code looks good! @bluefuton can you give it a rebase and then I'll test

**Comment by blowery** [comment]
code looks great, testing it out now

File: client/blocks/reader-email-settings/docs/example.jsx (hunk 3 above)
Comment (bluefuton) [inline client/blocks/reader-email-settings/docs/example.jsx]: I'm just mirroring the other use in `reader-main`, which also probably doesn't need it :)

File: client/devdocs/design/blocks.jsx (hunk 4 above)
Comment (bluefuton) [inline client/devdocs/design/blocks.jsx]: It passed! 🎉

File: client/blocks/reader-email-settings/docs/example.jsx (hunk 3 above)
Comment (bluefuton) [inline client/blocks/reader-email-settings/docs/example.jsx]: Removed 👍

**Pull request merged**